Source/WebCore/Modules/modern-media-controls/controls/vision-media-controls.css

```diff
@@ -314,17 +319,26 @@
     --slider-height: 20px;
 }
 
-.media-controls.vision:not(.audio) > .bottom.controls-bar,
-.media-controls.vision:not(.audio) > .bottom.controls-bar * {
+.media-controls.vision:not(.audio) .slider,
+.media-controls.vision:not(.audio) .slider * {
     transition: all 350ms;
 }
 
+.media-controls.vision:not(.audio) > .controls-bar.top-right *,
 .media-controls.vision:not(.audio) > .bottom.controls-bar * {
     transition-property: min-width, height, border-radius !important;
 }
 
+.media-controls.vision:not(.audio) > .controls-bar.simple-layout.top-right .fill:after {
+    --progress-bar-knob-margin: 1px;
+}
```

Comment:
Don't vision progress bars always `simple-layout` ?

Is this sufficient: `.media-controls.vision:not(.audio) > .controls-bar.top-right .fill:after` ?